src/main/service/case-service.ts

```diff
@@ -6,7 +6,7 @@ export function getCase(req, jid, ctid, cid) {
   const userId = req.authentication.user.id;
   const url = get("case_data_store_url") + "/caseworkers/" + userId + "/jurisdictions/" + jid + "/case-types/" + ctid +
     "/cases/" + cid;
-  const authorization = req.cookies.jwt ? `Bearer ${req.cookies.jwt}` : req.get(userReqAuth.AUTHORIZATION);
+  const authorization = req.cookies.jwt ? `Bearer ${req.cookies.jwt}` : req.headers[userReqAuth.AUTHORIZATION];
```

Comment:
I believe this change is the one causing the Data store to fail on request with a "bearer token missing" error.